lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -53,6 +56,13 @@ function WebProcessingServiceCatalogFunction(terria) {
      */
     this.identifier = undefined;
 
+    /**
+     * Gets or sets whether to use key value pairs (KVP) embedded in Execute URL, or whether to make a POST request
+     * with XML data.
+     * @type {Boolean}
+     */
+    this.useKvp = undefined;
```

Comment:
I agree that the default should be POST, but since TerriaJS has been defaulting to GET, if we change this it's going to break WPS for anyone whose WPS server can't accept a POST request (if such people exist).